src/zap.conf

```diff
@@ -31,6 +31,8 @@
 10027	OUTOFSCOPE	http://app:8080/public/js/uswds-init.min.js
 # get-gov.js contains suspicious word "from" as in `Array.from()`
 10027	OUTOFSCOPE	http://app:8080/public/js/get-gov.js
+# Ignore wording of "TODO"
```

Comment:
```suggestion
# Ignores suspicious word "TODO"
```
(Non-blocking) I'd recommend adding some additional context here (like above) as to what this addresses, though this is not an important change